components/identity/org.wso2.carbon.identity.application.authentication.endpoint/src/main/java/org/wso2/carbon/identity/application/authentication/endpoint/AuthenticationEndpoint.java

```diff
@@ -78,6 +78,10 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
                 loadPage = "passivests_login.do";
             } else if (request.getParameter("type").equals("oauth2") || request.getParameter("type").equals("oidc")) {
                 loadPage = "oauth2_login.do";
+            }else if (request.getParameter("type").equals("totp")) {
```

Comment:
Yes. Also in that case we can do a "comparator".equals(request.getParameter("type)) can't we ?. Will have to change all the code there